server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+require('dotenv').config();
+
+const app = express();
+const config = require('../webpack.config.js');
+
+const compiler = webpack(config);
+
+app.use(webpackDevMiddleware(compiler, {
+  publicPath: config.output.publicPath,
+}));
+
+const PORT = process.env.PORT || 8080;
+
+app.listen(PORT);
```

Comment:
index.js should only contain the listen method. Everything else should be imported from another file and the `app` exported for testing purposes.

Reply:
Refactored index.js to only listen, and created separate server.js file to create `app` server and export it.